src/index.js

```diff
@@ -1,10 +1,12 @@
 if (typeof window === 'undefined') {
-  global.crypto = {
-    async getRandomValues(b) {
-      const { randomFillSync } = await import('crypto');
-      randomFillSync(b);
-    }
-  };
+  if (!global.crypto) {
+    global.crypto = {
+      async getRandomValues(b) {
+        const { randomFillSync } = await import('crypto');
+        randomFillSync(b);
+      }
+    };
+  }
   global.performance = {
```

Comment:
I suggest to do this also for `global.performance`. Node.js already has `performance.now()`, so there should be no need to patch it in most cases.

Reply:
Good! Thanks for your advice!